simplified-app-ekirjasto/src/main/assets/txnative/fi/txstrings.json

```diff
@@ -196,7 +202,7 @@
          "string" : "Kirjaudu sisään pääsyavaimella"
       },
       "account_login_suomifi" : {
-         "string" : "Kirjaudu sisään Suomi.fi-tunnistuksella"
+         "string" : "Kirjaudu sisään suomi.fi:ssä"
```

Comment:
The original string (without this change) looks better to me? Should be fixed in Transifex (at least capitalized).